xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
```

Comment:
Another way to handle this might be to have a map which extracts the beamtime specific keys from the flat dictionaries.